source/string-key-of.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+Get string keys from type.
```

Comment:
Should be in sync with the readme.

Reply:
I think it should also explicitly mention that number keys are converted to strings. I don't think that's obvious. I first assumed it would **only** get the keys that are of string type.

Reply:
How about change description to "Convert a type which may have number keys to one with string keys, and get the string keys." ?

Reply:
You text does not say that the keys will be coerced. Use the readme text first and then do a second paragraph where it explicitly says that fact.